codec-http2/src/test/java/io/netty/handler/codec/http2/DefaultHttp2RemoteFlowControllerTest.java

```diff
@@ -756,6 +756,60 @@ public void reprioritizeShouldAdjustOutboundFlow() throws Http2Exception {
     }
 
     /**
+     * Test that the maximum allowed amount the flow controller allows to be sent is always fully allocated if
+     * the streams have at least this much data to send.
+     * <pre>
+     *         0
+     *        /|\
+     *       / | \
+     *      A  B  C
```

Comment:
How do we indicate this in the diagram? I think the `[B]` notation indicates the connection window is collapsed?